infra/charts/feast/charts/feast-core/values.yaml

```diff
@@ -72,8 +72,8 @@ application.yaml:
     stream:
       type: kafka
       options:
-        topic: TOPIC
-        bootstrapServers: HOST:PORT
+        topic: feast
+        bootstrapServers: kafka-events-client.gke-stage.postmates.net:9093
```

Comment:
this will be overwritten in prod?

Reply:
This will be a different value in prod, I'm going to use the environment.jsonnet for that to differentiate between deployments, but again this is for the helm, which we are not using